src/tasks/barfTasks.ts

```diff
@@ -114,7 +107,26 @@ function shouldGoUnderwater(): boolean {
   return have($effect`Fishy`) || (have($item`fishy pipe`) && use($item`fishy pipe`));
 }
 
-// Lights Out adventures require you to take several choices in a row
+type PrepOptions = {
+  requirements?: Requirement;
+  sea?: boolean;
+  location?: Location;
+};
+function prep(type: "free" | "meat", options: PrepOptions = {}): void {
```

Comment:
```suggestion
function prep(type: "free" | "embezzler", options: PrepOptions = {}): void {
```

I find it counterintuitive that you don't do meat prep for barf turn